sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -2528,9 +2528,218 @@
-              "optionsLabelPosition": "right",
+              "input": true,
               "lockKey": true
+            },
+            {
+              "label": "Which of the following best describes your situation? (If none of these apply you are not eligible for modified independant status)",
+              "optionsLabelPosition": "right",
+              "inline": false,
+              "tableView": false,
+              "values": [
+                {
+                  "label": " There is an irreconcilable rift between me and my parent(s), and I can provide satisfactory evidence of a severe and permanent breakdown in the relationship between you and your parent(s) that has existed for at least one year unless a sudden and serious situation has occurred.",
+                  "value": "irreconcilableRift",
+                  "shortcut": ""
+                },
+                {
+                  "label": " You would normally be classified as a dependant student by StudentAid BC, but you are unable to contact or locate your parents, step-parent, sponsor, or legal guardian due to conflict or emergency (such as war, famine or natural disaster) in the country where they live.",
+                  "value": "unableToContact",
+                  "shortcut": ""
+                },
+                {
+                  "label": " None of the above.",
+                  "value": "noneOfTheAbove",
+                  "shortcut": ""
+                }
+              ],
+              "validate": {
+                "required": true,
+                "onlyAvailableItems": true
+              },
+              "validateWhenHidden": false,
+              "key": "parentSituation",
+              "conditional": {
+                "show": true,
+                "when": "estranged",
+                "eq": "yes"
+              },
+              "type": "radio",
+              "input": true
+            },
+            {
+              "title": "Estranged details panel",
+              "collapsible": false,
+              "hideLabel": true,
+              "key": "estrangedDetailsPanel",
+              "customConditional": "show = ['irreconcilableRift', 'unableToContact'].includes(data.parentSituation);\r\n",
+              "type": "panel",
+              "label": "Estranged details panel",
+              "breadcrumbClickable": false,
+              "allowPrevious": false,
+              "buttonSettings": {
+                "previous": false,
+                "cancel": false,
+                "next": false
+              },
+              "scrollToTop": false,
+              "input": false,
+              "tableView": false,
+              "components": [
+                {
+                  "label": "Application Exception - Estranged from parents",
+                  "calculateValue": "value = \"studentApplicationException\";",
+                  "calculateServer": true,
+                  "key": "estrangedFromParentsApplicationException",
```

Comment:
There was a technical AC to rename the exception key to `modifiedIndependantApplicationException`. Was it not done on purpose to keep it simple?

Reply:
I had them flipped - can reverse this to use "modifiedIndependant..." instead.

Reply:
There is one AC that states

> Current exception path estranged from parents needs to be renamed to modified independent 

Is there a requirement to show different content while the Ministry is approving the exception?

Reply:
I updated the key and the content on the student exception form, as the terminology that the ministry seems to be using for this exception is "Modified Independent".

Reply:
Confirmed with business that the only thing that needs to change is the label for ministry users. I have reverted the key on both SFAA form and exceptions form.